Modules/Segmentation/KLMRegionGrowing/test/itkRegionGrow2DTest.cxx

```diff
@@ -1184,13 +1184,11 @@ test_regiongrowKLM2D()
       std::cout << "Test FAILED" << std::endl;
       if (itk::Math::NotAlmostEquals(pixelOut[0], pixelOutZero))
       {
-        std::cout << "pixelOut[0]: " << pixelOut[0] << " != "
-                  << "0" << std::endl;
+        std::cout << "pixelOut[0]: " << pixelOut[0] << " != " << '0' << std::endl;
```

Comment:
For what it's worth, it appears that Clang-Format has adjusted the formatting, just because `"0"` was replaced with `'0'`!

Reply:
Are you using clang-format version different from 8.0.2?

Reply:
```
ITK/clang-format-Windows.exe --version
clang-format version 8.0.0 (tags/RELEASE_800/final)
```
Looks different! But anyway, the CI clang-format linter appears to like this adjustment!

Reply:
Yet another one here, in my ITK build tree:

```
ITK/ClangFormat-prefix/src/ClangFormat/clang-format.exe --version
clang-format version 8.0.1 (tags/RELEASE_801/final)
```

Reply:
I thought that one in ITK is supposed to be 8.0.2. I am using this version (which says it is 8.0.1).